great_expectations/execution_engine/split_and_sample/sqlalchemy_data_splitter.py

```diff
@@ -526,19 +526,18 @@ def get_split_query_for_data_for_batch_identifiers_for_split_on_date_parts(
                         )
                     )
 
-                concat_clause: List[Label] = [
+                concat_clause = [
                     sa.func.distinct(concat_date_parts).label("concat_distinct_values"),
                 ]
             else:
-                concat_date_parts: concat = sa.func.concat(
+                concat_date_parts = sa.func.concat(
```

Comment:
It would be helpful to have this type declared above the `if`/`else` block in my opinion.